docs/user-guide/cai.rst

```diff
@@ -14,7 +16,7 @@ CAI allows training a harmless, but non-evasive AI assistant that engages with h
 .. _Constitutional AI (CAI): https://arxiv.org/abs/2212.08073
 
 CAI
-###############
+###
 The basic steps of CAI are described in this section and illustrated in the figure below (`Figure 1 <https://arxiv.org/abs/2212.08073>`_).
```

Comment:
please check the link. This link, (`Figure 1 <https://arxiv.org/abs/2212.08073>`_), goes to an arkiv article, not the Constitutitional AI Flow figure below.

Reply:
good catch. I've updated it to just say the figure below and the hyperlink is just to the anchor for the figure below.

Reply:
https://github.com/NVIDIA/NeMo-Aligner/pull/411/commits/9432333a44ad01d9534ed89263a37000b2aaca11